src/renderer/shared/ui/Inputs/Input/Input.tsx

```diff
@@ -6,16 +6,20 @@ import { HTMLInputProps } from '../common/types';
 import { CommonInputStyles, CommonInputStylesTheme } from '../common/styles';
 import { Theme } from '../../Dropdowns/common/types';
 
-export type Props = Pick<ComponentPropsWithoutRef<'input'>, HTMLInputProps> & {
+type NewType = {
   label?: ReactNode;
   invalid?: boolean;
   wrapperClass?: string;
   prefixElement?: ReactNode;
   suffixElement?: ReactNode;
   theme?: Theme;
   onChange?: (value: string) => void;
+  // @ts-ignore
+  onPaste?: (event: ClipboardEvent<HTMLInputElement>) => void;
```

Comment:
I've checked `ClipboardEvent` from `React` it satisfies TS
`import { ClipboardEvent } from 'react';`